src/celeritas/ext/detail/DetectorConstruction.hh

```diff
@@ -21,8 +21,8 @@ namespace detail
 class DetectorConstruction : public G4VUserDetectorConstruction
 {
   public:
-    explicit DetectorConstruction(G4String gdmlInput);
-    ~DetectorConstruction();
+    // Construct from a GDML filename
+    explicit DetectorConstruction(G4String gdml_input);
```

Comment:
I think we could drop the `G4String` in favor of `std::string`...

Reply:
Funnily enough I was going to, but then I looked into how `gdml_input` gets used, and it turns out `G4String` is not an alias of `std::string` but rather a *subclass* 😱 

Since this is a detail class that's specifically using the Geant4 interface maybe it's better to leave as is?

Reply:
Well I feel silly for changing it back to G4String then 😅  and I've now done as you suggest.